index.html

```diff
@@ -209,11 +224,15 @@ <h3>Skills</h3>
         </div>
     </section>
     <!-- Contact Form -->
-    <section class="contact">
+    <section class="contact" id="contact">
         <h2 class="contact_tle">
             I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </h2>
         <form action="https://formspree.io/f/xzbqzryz" method="POST" class="form">
+            <div class="desktop-name">
+                <input id="f-name" type="text" name="fName" placeholder="First Name" maxlength="30" required>
+                <input id="l-name" type="text" name="lName" placeholder="Last Name" maxlength="30" required>
+            </div>
```

Comment:
- [ ] Overall you did a great job working on this milestone! 🚀 However, according to the requirements, you should ignore the two inputs for first and last name and instead go with just one for the full name. This will keep it easy to handle the data. 🤓 🔧 

From the requirements:
![image](https://user-images.githubusercontent.com/77069011/213541943-5ba5a566-215c-447e-a74c-05298d2ea21a.png)